api/data_refinery_api/management/commands/post_downloads_summary.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+import requests
+
+from data_refinery_common.models import Dataset, DatasetAnnotation
+
+
+class Command(BaseCommand):
+    help = "Post downloads summary to slack"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--days",
+            type=int,
+            default=7,  # default to a week
+            help=("Number of days in the past for which to build the stats"),
+        )
+        parser.add_argument(
+            "--channel",
+            type=str,
+            default="ccdl-general",
+            help=("Optional parameter to choose the channel where the message will be posted."),
+        )
+
+    def handle(self, *args, **options):
+        """
+        """
```

Comment:
This docstring is empty. I feel like the help covers it though so this is fine and we can just remove these empty strings.